archon-ui-main/src/utils/taskOrdering.ts

```
@@ -0,0 +1,114 @@
+export function calculateTaskOrder(options: TaskOrderingOptions): number {
+  const { position, existingTasks, beforeTaskOrder, afterTaskOrder } = options;
+  
+  // Sort tasks by order for consistent calculations (without mutating input)
+  const sortedTasks = [...existingTasks].sort((a, b) => a.task_order - b.task_order);
+  
+  switch (position) {
+    case 'first':
+      if (sortedTasks.length === 0) {
+        return 65536; // Large seed value for better spacing
+      }
+      const firstOrder = sortedTasks[0].task_order;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)